### PR TITLE
Feature/534/checkbox list component

### PR DESCRIPTION
**Pull request by mxrcury**
image

### CODE REVIEW

File: src/stories/CheckboxList.stories.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import CheckboxList from '~/components/checkbox-list/CheckboxList'
+
+export default {
+  title: 'CheckboxList',
+  component: CheckboxList,
+  argTypes: {
+    items: [],
+    title: {
+      type: 'string',
```
Comment (MickhaelSh) [inline src/stories/CheckboxList.stories.jsx:9]: use ts and import interface from your component so you don't need write type manualy

Comment (mxrcury) [reply src/stories/CheckboxList.stories.jsx:9]: thx but as i know we dont need to write stories in ts

Comment (MickhaelSh) [reply src/stories/CheckboxList.stories.jsx:9]: it makes stories more informative, so it will be better if we will use it

Comment (mxrcury) [reply src/stories/CheckboxList.stories.jsx:9]: agree with you

File: src/components/app-select/AppSelect.tsx [hunk 2]
```diff
@@ -1,24 +1,32 @@
...
 
-const AppSelect = ({ setValue, value, fields, selectTitle = '' }) => {
+interface AppSelectProps {
+  setValue:(value:string) => void
+  value:string,
+  fields:SelectField[],
+  selectTitle?:string,
+}
+
+const AppSelect = ({ setValue, value, fields, selectTitle = '' }:AppSelectProps) => {
```
Comment (MickhaelSh) [inline src/components/app-select/AppSelect.tsx:18]: do as in CheckboxList with FC

File: src/components/app-select/AppSelect.tsx [hunk 3]
```diff
@@ -1,24 +1,32 @@
...
   const fieldsList = fields.map(field => (
     <MenuItem key={ field.value } value={ field.value }>
       { t(field.title) }
     </MenuItem>)
   )
   const titleEl = selectTitle.length ? (
-    <Typography sx={ styles.selectTitle } variant='subtitle1' >
+    <Typography aria-label='select-title'  sx={ styles.selectTitle } variant='subtitle1' >
       { t(selectTitle) }
       :
```
Comment (MickhaelSh) [inline src/components/app-select/AppSelect.tsx:31]: move this to translation consts

Comment (mxrcury) [reply src/components/app-select/AppSelect.tsx:31]: aria-label?

File: src/components/app-select/AppSelect.tsx (hunk 2 above)
Comment (mxrcury) [reply src/components/app-select/AppSelect.tsx]: ok, changed

File: src/components/app-select/AppSelect.tsx (hunk 3 above)
Comment (MickhaelSh) [reply src/components/app-select/AppSelect.tsx:32]: ':'

**Review by maarekua — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

File: src/components/app-select/AppSelect.tsx (hunk 3 above)
Comment (mxrcury) [reply src/components/app-select/AppSelect.tsx]: fixed

**Pull request merged**